client/blocks/inline-help/contextual-help.js

```diff
@@ -52,700 +59,795 @@ const contextLinksForSection = {
-				'all in one place.',
+			title: translate( 'The Media Library' ),
+			description: translate(
+				'The Media Library is where you can manage your images, audio, videos, and documents all in one place.'
+			),
 		},
 		{
 			link: 'http://en.support.wordpress.com/add-media/finding-free-images-and-other-media/',
 			post_id: 78425,
-			title: 'Finding Free Images and other Media',
-			description:
-				'Use free images (and other media) to make your pages and posts more interesting and engaging when on a budget!',
+			title: translate( 'Finding Free Images and other Media' ),
+			description: translate(
+				'Use free images (and other media) to make your pages and posts more interesting and engaging when on a budget!'
+			),
 		},
 		{
 			link: 'http://en.support.wordpress.com/add-media/',
 			post_id: 38830,
-			title: 'Add Media',
-			description:
+			title: translate( 'Add Media' ),
+			description: translate(
 				'Dress up your text-based posts and pages with individual images, image galleries, ' +
-				'slideshows, videos, and audio.',
+					'slideshows, videos, and audio.'
+			),
 		},
 		{
 			link: 'http://en.support.wordpress.com/videos/',
 			post_id: 4744,
-			title: 'Including Videos',
-			description:
-				'Videos are a great way to enhance your site pages and blog posts. Learn how to include them.',
+			title: translate( 'Including Videos' ),
+			description: translate(
+				'Videos are a great way to enhance your site pages and blog posts. Learn how to include them.'
+			),
 		},
 	],
 	people: [
 		{
 			link: 'http://en.support.wordpress.com/user-roles/',
-			title: 'User Roles',
-			description:
-				'User roles determine the access level or permissions of a person authorized to use a WordPress.com site.',
+			title: translate( 'User Roles' ),
+			description: translate(
+				'User roles determine the access level or permissions of a person authorized to use a WordPress.com site.'
+			),
 		},
 		{
 			link: 'http://en.support.wordpress.com/user-mentions/',
 			post_id: 91788,
-			title: 'User Mentions',
-			description:
+			title: translate( 'User Mentions' ),
+			description: translate(
 				'User mentions are a great way to include other WordPress.com users within your ' +
-				'posts and comments.',
+					'posts and comments.'
+			),
 		},
 		{
 			link: 'http://en.support.wordpress.com/adding-users/',
 			post_id: 2160,
-			title: 'Inviting Contributors, Followers, and Viewers',
-			description:
-				'Invite contributors, followers, and viewers to collaborate with others and grow your audience!',
+			title: translate( 'Inviting Contributors, Followers, and Viewers' ),
+			description: translate(
+				'Invite contributors, followers, and viewers to collaborate with others and grow your audience!'
+			),
 		},
 		{
 			link: 'http://en.support.wordpress.com/followers/',
 			post_id: 5444,
-			title: 'Your Followers',
-			description:
+			title: translate( 'Your Followers' ),
```

Comment:
:disappointed: 56 existing translations will be lost with this change.
If the string must change, the following option has already been translated **43** times:
`translate( 'followers' )` *ES Score: 10*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/localized-support-urls-inline-contextual-help)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).